documentation/api/io.strimzi.api.kafka.model.kafka.tieredstorage.TieredStorageCustom.adoc

```diff
@@ -0,0 +1,20 @@
+To configure the tiered storage feature, set the `type` property to `custom`.
+
+Custom tiered storage allows for custom RemoteStorageManager configuration to be used. By default Strimzi will use the `TopicBasedRemoteLogMetadataManager` for RLMM configuration.
```

Comment:
Can you add here that the user needs to add the tiered storage plugin tot he image first by building a custom container image?

Reply:
Done